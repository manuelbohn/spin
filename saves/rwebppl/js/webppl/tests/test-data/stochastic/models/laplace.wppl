var model = function() {
  return laplace(0, 5 / Math.sqrt(2));
};
