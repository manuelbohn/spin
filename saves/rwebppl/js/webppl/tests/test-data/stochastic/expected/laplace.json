{
  "mean": 0,
  "std": 5
}
